crates/compiler/mono/src/ir.rs

```diff
@@ -1675,35 +1676,13 @@ pub enum BranchInfo<'a> {
 }
 
 impl<'a> BranchInfo<'a> {
-    pub fn to_doc<'b, D, A>(&'b self, alloc: &'b D) -> DocBuilder<'b, D, A>
+    pub fn to_doc<'b, D, A>(&'b self, alloc: &'b D, _pretty: bool) -> DocBuilder<'b, D, A>
     where
         D: DocAllocator<'b, A>,
         D::Doc: Clone,
         A: Clone,
     {
-        use BranchInfo::*;
-
-        match self {
-            Constructor {
-                tag_id,
-                scrutinee,
-                layout: _,
-            } if pretty_print_ir_symbols() => alloc
-                .hardline()
-                .append("    BranchInfo: { scrutinee: ")
-                .append(symbol_to_doc(alloc, *scrutinee))
-                .append(", tag_id: ")
-                .append(format!("{}", tag_id))
-                .append("} "),
-
-            _ => {
-                if pretty_print_ir_symbols() {
-                    alloc.text(" <no branch info>")
-                } else {
-                    alloc.text("")
-                }
-            }
-        }
+        alloc.text("")
```

Comment:
I do think we'll benefit long-term from storing/displaying more info about pattern matches/switches. Fine for now though